### ISSUE TITLE
CUDA_ERROR_NO_BINARY_FOR_GPU

### ISSUE THREAD

**Issue by Domi-Z** [asadprodhan/GPU-accelerated-guppy-basecalling#2]
Hi，
Thank you for sharing,  it's really help me a lot. 

The following error occurs when I use the -x parameter:
`[guppy/error] *Common::LoadModuleFromFatbin: Loading fatbin file shared.fatbin failed with: CUDA error at /builds/ofan/ont_core_cpp/ont_core/common/cuda_common.cpp:54: CUDA_ERROR_NO_BINARY_FOR_GPU`

Do you have any idea how to fix this? 
Best

**Comment by Domi-Z** [comment]
The command I entered was:
`../../00_biosoft/ont-guppy/bin/guppy_basecaller -i ../../02_WA_fast5/02_out_single_fast5/ -r -s ./ --flowcell FLO-PRO002 --kit SQK-LSK109 --disable_pings -x 'cuda:0' --min_qscore 7 --num_callers 4 --gpu_runners_per_device 8`

My linux version number is Linux version 4.15.0-209-generic (buildd@lcy02-amd64-091) (gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04))
My CUDA Driver is 465.19.01, i have two 3090 GPU.
Let me know if you need any more info.

**Comment by Domi-Z** [comment]
Hi，
Don't bother about that.
I found the solution in this[ link](https://forums.developer.nvidia.com/t/cuda-error-while-loading-fatbin-guppy/128743).
The issue was that it was built under Ubuntu 16.04 for the TX2.
ONT fixed the issue in a new version of Guppy (4.0.14). And my Guppy version is 3.6.1, This problem was solved when I installed an newer version of Guppy. 
Anyway, thanks again for your sharing!
